### DISCUSSION TITLE
I can't update my wallet

### DISCUSSION THREAD

**Discussion by FedeCosmico** [iotaledger/firefly#7898] [Help & Support]
It asks me to update and restart again and again and it never finishes updating. Does anyone know what can be done? Thank you!

**Comment by begonaalvarezd** [comment]
You can try updating by downloading the latest version directly from the official website https://firefly.iota.org/

**Reply by FedeCosmico** [reply]
Gracias por tu respuesta. Probe de bajar otra billetera, la puedo abrir normalmente pero mi anterior billetera sigue sin poder ser actualizada. Puedo eliminarla y quedarme con esta nueva?

**Reply by begonaalvarezd** [reply]
Si claro, si te funciona la nueva puedes eliminar la anterior
